### DISCUSSION TITLE
Danke für vlmcsd

### DISCUSSION THREAD

**Discussion by manfred-mueller** [Freetz-NG/freetz-ng#1088] [Allgemein] [answered]
Ich möchte einmal DANKE an fda77 sagen für die Aufnahme und 'Professionalisierung' von vlmcsd.
Beim Durchgehen deiner Änderungen musste ich erkennen, dass ich die Paketierung in freetz-ng nicht wirklich verstanden hatte ;-)
Es lief zwar auch mit der 'Setup-Krücke' vlmcsdsetup einwandfrei, aber so, wie es jetzt ist, ist es eine perfekte und vor Allem 'saubere' Lösung. Ich habe es mit 7490/7590 gestetet: DANKE!
p.s. 10$ sind nicht viel aber ein kleines Dankeschön

**Comment by fda77** [comment, marked as answer]
Danke! Das ist mehr als von 99,9% der anderen!

**Comment by kainers1** [comment]
Danke auch von mir. Dachte mir vor einigen Jahren schon mal, dass das cool wäre, aber ich bin dazu zu unbegabt, sodass es bis jetzt in einem Dockercontainer lief.

Dem AVM-Dhcp [SRV-Einträge](https://github.com/SystemRage/py-kms/issues/121#issuecomment-782776076) beibringen wird vermutlich nicht möglich sein und es bleibt nur dnsmasq für diesen Zweck?

**Reply by pfichtner** [reply]
Ja, man nehme [dnsmasq](https://freetz-ng.github.io/freetz-ng/make/dnsmasq.html)
